### PR TITLE
removed legacy objectFit property from next/image component

### PR DESCRIPTION
**Pull request by Nkiriobasi**
Closes #{issue number}

## Screenshots:

Before|After
---|---
Paste screenshot|Paste screenshot

### New environment variables:

- [ ] `NEW_ENV_VAR`: env var details

### New or updated dependencies:

Dependency name|Previous version|Updated version|Details
---|---|---|---
`dependency/name`|`v1.0.0`|`v2.0.0`|

### CODE REVIEW

File: src/components/client/about/AboutPage.styled.tsx [hunk 1]
```diff
@@ -15,7 +15,6 @@ export const AboutHeading = styled(Typography)(() => ({
 export const Avatar = styled(Image)(() => ({
   borderRadius: '50%',
   textAlign: 'center',
-  objectFit: 'cover',
```
Comment (sashko9807) [inline src/components/client/about/AboutPage.styled.tsx:18]: Should be reverted.

File: src/components/client/blog/FeaturedImage.tsx [hunk 2]
```diff
@@ -10,14 +10,14 @@ const defaultImage = {
 type Props = {
   height: string
   src?: string | null
-  objectFit?: 'fill' | 'contain' | 'cover' | 'none' | 'scale-down'
+  // objectFit?: 'fill' | 'contain' | 'cover' | 'none' | 'scale-down'
```
Comment (sashko9807) [inline src/components/client/blog/FeaturedImage.tsx:13]: Should be reverted

File: src/components/client/blog/FeaturedImage.tsx [hunk 3]
```diff
@@ -29,7 +29,7 @@ export default function FeaturedImage({
         <Image
           fill
           sizes="100vw"
-          style={{ objectFit, objectPosition: defaultImage.objectPosition }}
+          style={{ objectPosition: defaultImage.objectPosition }}
```
Comment (sashko9807) [inline src/components/client/blog/FeaturedImage.tsx:32]: Should be reverted

File: src/components/client/campaigns/CampaignSlider.tsx [hunk 4]
```diff
@@ -82,7 +82,7 @@ export default function CampaignSlider({ sliderImages }: Props) {
       <Root className={classes.container}>
         <div style={{ textAlign: 'center' }}>
           {/* A11Y TODO: Add proper alt text*/}
-          <Image src={sliderImages[0]} alt="campaign" height={300} width={400} objectFit="cover" />
+          <Image src={sliderImages[0]} alt="campaign" height={300} width={400} />
```
Comment (sashko9807) [inline src/components/client/campaigns/CampaignSlider.tsx:85]: This is ok

File: src/components/client/campaigns/CampaignSlider.tsx [hunk 5]
```diff
@@ -96,7 +96,7 @@ export default function CampaignSlider({ sliderImages }: Props) {
         {sliderImages.map((image, index) => (
           <div key={index}>
             {/* A11Y TODO: Add proper alt text*/}
-            <Image src={image} alt="campaign" height={300} width={400} objectFit="cover" />
+            <Image src={image} alt="campaign" height={300} width={400} />
```
Comment (sashko9807) [inline src/components/client/campaigns/CampaignSlider.tsx:99]: Move `objectFit='cover'` inside the style prop

File: src/components/client/index/sections/TeamMembersSection/TeamMembersSection.tsx [hunk 6]
```diff
@@ -21,7 +21,7 @@ export default function TeamMembersSection() {
         <Image
           alt="Team image"
           src={teamImagePath}
-          style={{ maxWidth: '100%', height: 'auto', objectFit: 'contain' }}
+          style={{ maxWidth: '100%', height: 'auto' }}
```
Comment (sashko9807) [inline src/components/client/index/sections/TeamMembersSection/TeamMembersSection.tsx:24]: Should be reverted.

**Review by sashko9807 — CHANGES_REQUESTED** [review]
Please keep in mind that, objectFit is a valid CSS property. What the warning from issue #1529 refers to is move the objectFit, inside the style of the image component, rather than it being a prop.

So if  we have `<Image ..... objectFit='contain'/>`
We need to move objectFit into style prop e.g.
`<Image style={{objectFit: 'contain}} />`

**Comment by Nkiriobasi** [comment]
Alright, no problem

**Comment by ani-kalpachka** [comment]
@Nkiriobasi thanks for the contribution!
I think we should keep the objectFit: 'cover' because without it the styles do not look appropriate.

**Review by igoychev — APPROVED** [review]
thanks for the help!

**Review by ani-kalpachka — APPROVED** [review]

**Pull request merged**